description: |-
  It was discovered that aufs improperly managed inode reference counts in the
  vfsub_dentry_open() method. A local attacker could use this vulnerability to
  cause a denial of service attack.
references:
- https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-11935
- https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1873074
- https://usn.ubuntu.com/usn/usn-4425-1
- https://usn.ubuntu.com/usn/usn-4426-1
- https://usn.ubuntu.com/usn/usn-4427-1
- https://usn.ubuntu.com/usn/usn-4439-1
- https://usn.ubuntu.com/usn/usn-4440-1
- https://usn.ubuntu.com/lsn/0070-1/
- https://ubuntu.com/security/notices/USN-4425-1
- https://ubuntu.com/security/notices/USN-4426-1
- https://ubuntu.com/security/notices/USN-4427-1
- https://ubuntu.com/security/notices/USN-4439-1
- https://ubuntu.com/security/notices/USN-4440-1
comments:
  ubuntu/sbeattie: |-
    upstream aufs4-linux.git commits:
      515a586eeef31e0717d5dea21e2c11a965340b3c
      f10aea57d39d6cd311312e9e7746804f7059b5c8
reporters:
- Mauricio Faria de Oliveira
introduced-by:
  mainline: never
fixed-by:
  ubuntu/bionic: [ecd76488aa0e34edda1e7285e075c27f8ce22091, 291d96db24358bd46689998bc0c0a09ee7905666]
  ubuntu/focal: [4d4c05a8901e313264433db69cc3374cef5164f6]
  ubuntu/xenial: [f500cf60dc34e0632ea9de96d24406f84fe66030, 332c268dea5d02f04fdf27837bc2e712e0b99aa2]
